commit c022802bb4cc73608c984e7bd0875dc1e44d2ca2
Author: Daniel Nicoletti <dantt12@gmail.com>
Date:   Tue Jun 19 13:43:51 2012 -0300

    Libsolid now reports the right valeu when it should conserve resources

diff --git a/apperd/ApperdThread.cpp b/apperd/ApperdThread.cpp
index 6c270e6..c56dff6 100644
--- a/apperd/ApperdThread.cpp
+++ b/apperd/ApperdThread.cpp
@@ -314,8 +314,8 @@ bool ApperdThread::isSystemReady(bool ignoreBattery, bool ignoreMobile) const
     // check how applications should behave (e.g. on battery power)
     if (!ignoreBattery && Solid::PowerManagement::appShouldConserveResources()) {
         kDebug() << "System is not ready, application should conserve resources";
-        // FIXME this always return true even on AC
-//        return false;
+        // This was fixed for KDElibs 4.8.5
+        return false;
     }
 
     // TODO it would be nice is Solid provided this
